steve/util.py

```diff
@@ -216,18 +391,21 @@ def get_project_config_file_name():
     return SteveConfig.file_name
 
 
-def create_project_config_file(dir_path):
+def create_project_config_file(dir_path, cfg=None):
     """Creates a new config file in directory
 
     :param dir_path: directory in which to create the config file
+    :param cfg:      if set alternative from default SteveConfig
 
     :returns: config file basename
     """
-    SteveConfig().create(dir_path)
+    if cfg is None:
+        cfg = SteveConfig
+    cfg().create(dir_path)
 
 
 def get_from_config(cfg, key, section='project',
-                    error='"{key}" must be defined in steve.ini file.'):
+                    error='"{key}" must be defined in {name} file.'):
```

Comment:
I looked through the code and the git history and have no idea why I made this pass in the error message. We should nix that as an argument.